demo/demo.js

```diff
@@ -23,15 +25,21 @@ document.querySelectorAll(".categories > li").forEach((category) => {
   });
 });
 
-document.querySelectorAll(".styles > li").forEach((style) => {
+document.querySelectorAll(".styles a").forEach((style) => {
```

Comment:
```suggestion
document.querySelectorAll(".styles li a").forEach((style) => {
```

More descriptive as these are still a list.

Reply:
Or perhaps just `ul.styles` ?

Reply:
Changed to your first suggestion.